### DISCUSSION TITLE
First configuration domain check not works

### DISCUSSION THREAD

**Discussion by temuccio** [nextcloud/all-in-one#4041] [Questions] [answered]
Hi all, 
on my Debian 12, I am trying to run this Docker container:

```
docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest
```

Next, I am trying to configure the domain in the Nextcloud master container. I have a public IP and a subdomain. In fact, to configure the master container, I access it from:

`https://my.domain.com:8443`

However, when I submit the domain, I get the following error:

`"The server is not reachable on Port 443. You can verify this e.g. with 'https://portchecker.co/' by entering your domain there as ip-address and port 443 as port."`

There isn't a problem with the firewall because if I start the Apache server on Debian, I can see the page from the domain. I tried to ping (or open with curl) my.domain.com on the shell on the Debian server, but it did not respond. Is this the problem? Do I need to "pre-configure" the Debian system?
I try to check if the port is open and it works.
```
root@docker:~# netstat -tulpn | grep :443
tcp        0      0 0.0.0.0:443             0.0.0.0:*               LISTEN      1205741/docker-prox
tcp6       0      0 :::443                  :::*                    LISTEN      1205749/docker-prox`
```
From debian server, I try this:
```
root@docker:~# openssl s_client -quiet -connect localhost:443
140040934290752:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../ssl/record/ssl3_record.c:331:
root@docker:~# openssl s_client -quiet -connect my.domain.com:443
connect:errno=110
```

**Comment by szaimen** [comment, marked as answer]
> I tried to ping (or open with curl) my.domain.com on the shell on the Debian server, but it did not respond. Is this the problem?

Yes, it is. Also see https://github.com/nextcloud/all-in-one/pull/4046
